### PR TITLE
Test - Working on Bugs

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://docs.google.com/document/d/1vbtojbw9-Mq5I46P47TYpzQ9auKWYHxeRa9eZTGKqdI/edit?usp=sharing

adjustment popup text pending

### CODE REVIEW

File: apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts [hunk 1]
```diff
@@ -369,4 +363,14 @@ export class RatioAnalysisChartComponent implements OnInit, OnDestroy {
         this.destroyed$.next(true);
         this.destroyed$.complete();
     }
+
+    public translationComplete(event: boolean): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts:367]: Add JS doc

File: apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts [hunk 2]
```diff
@@ -369,4 +363,14 @@ export class RatioAnalysisChartComponent implements OnInit, OnDestroy {
...
+        if(event) {
+            this.rationResponse$.pipe(skipWhile(p => (isNullOrUndefined(p)))).subscribe(p => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts:369]: Add takeUntil, avoid single character name

File: apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts [hunk 3]
```diff
@@ -428,6 +428,7 @@ export class ReportsDetailsComponent implements OnInit, OnDestroy {
         if(event) {
             this.monthNames = [this.commonLocaleData.app_months_full.january, this.commonLocaleData.app_months_full.february, this.commonLocaleData.app_months_full.march, this.commonLocaleData.app_months_full.april, this.commonLocaleData.app_months_full.may, this.commonLocaleData.app_months_full.june, this.commonLocaleData.app_months_full.july, this.commonLocaleData.app_months_full.august, this.commonLocaleData.app_months_full.september, this.commonLocaleData.app_months_full.october, this.commonLocaleData.app_months_full.november, this.commonLocaleData.app_months_full.december];
             this.setCurrentFY();
+            this.getSelectedDuration();
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts:431]: This method returns a string which is unused

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts:431]: @prateek-giddh this is being used in html file.

File: apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts]: takeUntil is already available in constructor.

File: apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts (hunk 1 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts:373]: Done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**